cmf-cli/Constants/CliConstants.cs

```diff
@@ -1,9 +1,26 @@
+using System.Management.Automation.Runspaces;
+using System.Security.Authentication.ExtendedProtection;
 using System.Text.RegularExpressions;
 
 namespace Cmf.CLI.Constants
 {
     public static class CliConstants
     {
+        /// <summary>
+        /// The package name used for INPMClient implementations
```

Comment:
There not much point saying where this is used (I have Find References), but what the actual use of these variables is

Reply:
I have removed these constants, doesn't make sense anymore